panelapp/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.db.models import JSONField
+
+from seqr.models import LocusList as SeqrLocusList, LocusListGene as SeqrLocusListGene
+
+
+class PaLocusList(models.Model):
+    """PanelApp extension of seqr.models.LocusList."""
```

Comment:
add a blank line before the docstring

Reply:
Sorry, I must be missing something.  Are there more details on how to pass these style guides?  I've tried many combinations of formatting styles and can't see to get Codacy happy.  Indeed, are there conflicts with some of these pydocstyle rules?  https://github.com/PyCQA/pydocstyle/issues/141

Reply:
I'm sorry, our codacy rules for docstrings are a nightmare. If that round of adjustments didn't fix the error, we can merge it with the codacy issues and I'll get them cleaned up later

Reply:
Just to satisfy my curiosity, what were the Codacy changes required to make it happy?  Not urgent of course, it's more for my future reference.